src/5.Functions/functions.php

```diff
@@ -21,8 +21,8 @@ function nickname_generate($seed)
 function object_generate()
 {
     // Create stuff
-    $anArray = ['Seed1", "Seed2", "Seed3", "Seed4'];
-    $anAssocArray = ['AASeed1" => "Seed1", "AASeed2" => "Seed2", "AASeed3" => "Seed3", "AASeed4" => "Seed4'];
+    $anArray = ["Seed1", "Seed2", "Seed3", "Seed4"];
```

Comment:
Indeed, quotes were mismatched here, probably the result of a erroneous search and replace.